Automattic/wp-calypso
# Inline Help: Add Rich Results

## This PR
- aims to add Rich Results (Link + Description, Video, Guided Tour) to Inline Help

**Note:** `translate` calls omitted as the help articles themselves aren't translated either.


## How to test

- Open the Inline Help by clicking the blue question mark in the bottom right
- Click one of the results
- A Rich Result should open
- Clicking the Button in the Rich Result should either: 
  - Open a help article in place (article)
  - Open a dialog and play a video (video)
  - Start a guided tour (tour)
- To test a tour go to `Media` and select the first item in the results list
- To test a tour go to `Sharing` and select the first item in the results list

## What to test
- Ensure everything is working as expected  (see GIFs below)
- Ensure that the right Tracks Events are fired for each interaction
- Does it work on Browsers other than Chrome?
- Does it work on mobile?

## What it should look like

### Video
![video](https://user-images.githubusercontent.com/1562646/39307936-16763f4a-495c-11e8-8493-c2dec278a70a.gif)


### Tour
![tour](https://user-images.githubusercontent.com/1562646/39307839-d82b624c-495b-11e8-9a19-c8c13e3642e4.gif)
